docs/fhir-guide/FHIR 101 - Practical Guide.ipynb

```diff
@@ -0,0 +1,2966 @@
+   "source": [
+    "## Tutorial"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "### Real World Use Case\n",
+    "In this tutorial we will create a simple data model that is able to capture basic demographic information about Leukemia patients that have had a biosample drawn from them.  No protected health information [(PHI)](https://www.hipaajournal.com/considered-phi-hipaa/) data should be captured. In fact there should be no PHI attributes available anywhere in the model.\n",
+    "\n",
+    "At a minimum we want to capture:\n",
+    "\n",
+    "**Patient Attributes**\n",
+    "- Gender (required)\n",
+    "- Race (required)\n",
+    "- Biological Sex (required)\n",
+    "\n",
+    "**Specimen Attributes**\n",
+    "- The patient from whom the sample was drawn\n",
+    "- Anatomical site or location on the body from which the sample was drawn\n",
+    "- Composition - the type of sample - blood, tissue, saliva, etc.\n",
+    "\n",
+    "### What You'll Learn\n",
+    "You will learn the following practical concepts in this tutorial:\n",
+    "\n",
+    "- How to create a new model for a FHIR resource by extending it's base model\n",
+    "- How to add/remove attributes from the base model\n",
+    "- How to add make attributes required\n",
+    "- How to make an attribute searchable\n",
+    "- How to add a coded attribute - or one that is restricted to an enumeration defined by an ontology\n",
+    "- How to validate your model\n",
+    "- How to document your model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "### Setup"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "heading_collapsed": true,
+    "hidden": true
+   },
+   "source": [
+    "#### Requirements\n",
+    "**FHIR Server**\n",
+    "\n",
+    "- We will use [HAPI](http://hapi.fhir.org/), the reference FHIR server, for this tutorial. \n",
+    "\n",
+    "**Base URL**\n",
+    "\n",
+    "- All HTTP requests will be sent to HAPI's base URL: http://hapi.fhir.org/baseR4\n",
+    "\n",
+    "**Model Development Editor**\n",
+    "\n",
+    "You will need something to author JSON formatted model files. Unfortunately there are almost no user friendly tools for this. Most model development is done in a code editor. This can be difficult as it requires the developer to unecessarily remember complex syntax of the FHIR DSL in order to create and edit model files.\n",
+    "\n",
+    "- **Forge** - is a [visual editor](https://simplifier.net/forge) for FHIR model development.     It does make some things easier, but it can be buggy and was natively developed for windows.   It is possible to [run it on OSX](http://docs.simplifier.net/forge/forgeInstall.html#running-on-macos). You can use it, but       explanation of how to use it is out   of the scope of this tutorial."
```

Comment:
```suggestion
    "- **Forge** - is a [visual editor for FHIR model development](https://simplifier.net/forge). It does make some things easier, but it can be buggy and was natively developed for Windows. It may be possible to [run Forge on macOS using Wine](http://docs.simplifier.net/forge/forgeInstall.html#running-on-macos)."
```